aeolic.go

```diff
@@ -20,7 +22,8 @@ type Client struct {
 
 // New - returns a new client with the templates loaded from the provided directory.
 func New(apiKey string, templateDir string) (Client, error) {
-	templates, err := withTemplates(templateDir, ".tmpl.json")
+	fsys := os.DirFS("./")
```

Comment:
Could actually do this now.

```suggestion
	fsys := os.DirFS("./")
	return NewWithFS(apiKey, fsys, templateDir)
```